spot_wrapper/calibration/calibration_util.py

```diff
@@ -191,9 +196,10 @@ def get_multiple_perspective_camera_calibration_dataset(
                     os.path.join(data_path, str(jdx), f"{idx}.png"),
                     image,
                 )
+            np.save(os.path.join(data_path, "poses", f"{idx}.npy"), new_pose)
     calibration_images = np.array(calibration_images, dtype=object)
     auto_cam_cal_robot.shutdown()
-    return np.array(calibration_images, dtype=object)
+    return (np.array(calibration_images, dtype=object), poses)
```

Comment:
Do you need to reset this as an `np.array` if it's already set as that type on line 199?